.gh-pages-update.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+rm -rf out || exit 0;
+rm -rf master.zip
+rm -rf pander
+rm -rf pander-master
+
+GH_REPO="@github.com/RomanTsegelskyi/pander.git"
```

Comment:
Yes, I will do small updated right before merge. Since I have write access to repo, my token should work fine